src/test_utils.rs

```diff
@@ -27,6 +27,7 @@ pub(crate) fn make_issuer(
     rcgen::Certificate::from_params(ca_params).unwrap()
 }
 
+#[cfg_attr(not(feature = "ring"), allow(dead_code))]
```

Comment:
I think with the refactoring I did in https://github.com/rustls/webpki/pull/178 this will fall away because `end_entity.rs`'s `printable_string_common_name` test will use the `make_end_entity` helper from a test that doesn't require `ring` :crossed_fingers: